### ISSUE TITLE
New tab could stuck when zsh is waiting for user-input.

### ISSUE THREAD

**Issue by haitch** [warpdotdev/Warp#533]
### Describe the bug

```shell
new tab stuck for a while until I type Y in another terminal window.


[oh-my-zsh] Would you like to update? [Y/n]
```

### To Reproduce

Steps to reproduce:
2. Click on '...'
3. Scroll down to '...'
4. See error '...'

**Comment by elviskahoro** [comment]
Thanks for catching this. If you can share you RC files here that'd be helpful, but if not let'
s discuss over email: elvis@warp.dev

**Comment by elviskahoro** [comment]
@haitch we're trying to reproduce it, can you share some steps or a video? It would be really helpful. Thanks in advance.

**Comment by kalenpw** [comment]
I am having the same issue, basically if there is an oh-my-zsh update, Warp hangs since it is waiting for the user to enter y/n on the update.  The initial window will never load, but I can open a new tab which loads zsh (I believe at this point it just ignores the update)

I tried entering either and it doesn't allow for input. Attached is a screenshot

Here is a gist with my .zshrc, it is just the default oh-my-zsh one: https://gist.github.com/kalenpw/f120a373dc773faa8c0509a0564a647f

As far as reproducing it goes, I can't really think of a way to besides just installing oh-my-zsh and waiting for an update (they seem to happen at least weekly).  If there is any info I can grab off my computer to help debug it let me know. 

(As an aside I started using Warp last week and have been enjoying it, thanks for the great software!)

![Screen Shot 2022-01-10 at 11 03 42](https://user-images.githubusercontent.com/13550971/148816380-c6c55fb9-ddba-4260-a423-ac2bf49c39f5.png)
